src/LabelTrack.h

```diff
@@ -88,9 +88,23 @@ class AUDACITY_DLL_API LabelTrack final
    , public wxEvtHandler
 {
  public:
+   static wxString GetDefaultName();
+
    // Construct and also build all attachments
    static LabelTrack *New(AudacityProject &project);
 
+   /**
+    * \brief Creates a new LabelTrack with specified @p name and appends it to the @p trackList
+    * \return New LabelTrack with custom name
+    */
+   static LabelTrack* Create(TrackList& trackList, const wxString& name);
+
+   /**
+    * \brief Create a new LabelTrack with default name and appends it to the @p trackList
+    * \return New LabelTrack with default name
+    */
+   static LabelTrack* Create(TrackList& trackList);
```

Comment:
Comment the two functions above about the intention to add unique names, unless that is caller's responsibility.  Or maybe add a boolean to control whether there is unique numbering.

Reply:
Only one with default name assigns unique name to the track